pkg/services/meta/meta.go

```diff
@@ -26,14 +28,18 @@ const (
 	rootKey = 0x00
 )
 
-// ContainerLister is a source of actual containers current node belongs to.
-type ContainerLister interface {
+// NeoFSNetwork describes current NeoFS storage network state.
+type NeoFSNetwork interface {
 	// List returns node's containers that support chain-based meta data and
 	// any error that does not allow listing.
 	List() (map[cid.ID]struct{}, error)
 	// IsMineWithMeta checks if the given CID has meta enabled and current
 	// node belongs to it.
 	IsMineWithMeta(cid.ID) (bool, error)
+	// Head returns actual object header from the NeoFS network (non-local
+	// objects should also be returned). Missing, removed object statuses
+	// must be reported according to API statuses from SDK.
```

Comment:
this statement is not needed actually, `Meta` does not care what error is returned

Reply:
imaginary logic makes the interface heavier without any practical benefit